### PR TITLE
Refactor styled components

### PR DESCRIPTION
**Pull request by bagshik**
### Issue #43

Moved all styled components into their own file 👌

### CODE REVIEW

File: src/button/button-bar.jsx [hunk 1]
```diff
@@ -1,20 +1,9 @@
...
-`;
-const FirstRow = styled.div`
-  color: ${({ theme }) => `${theme.inverted}`};
-  display: flex;
-`;
-const SecondRow = styled.div`
-  color: ${({ theme }) => `${theme.inverted}`};
-  display: flex;
-`;
+import Main from './main.styled';
```
Comment (EricTurf) [inline src/button/button-bar.jsx]: Using this as an example but for any component using multiple styled components could you make __one__ `.styled.js` file and name export the multiple styled components

So in this case
`button-bar.styled.js`

```javascript
export const Main=styled.div` ... `
export const FirstRow = styled.div ` ..`
export const SecondRow = styled.div ` ..`
```

`button-bar.jsx````javascript
import{ Main, FirstRow, SecondRow } from './button-bar.styled
```

Once thats done this is awesome :)

Comment (bagshik) [reply src/button/button-bar.jsx]: Aight, I'm on it.

**Comment by bagshik** [comment]
Aight I'm done, this should do it I think?

**Comment by EricTurf** [comment]
Yep, looks good to me. Thanks for your work

**Pull request merged**